app/views/quotations/show.html.erb

```diff
@@ -1,7 +1,17 @@
 <h1>Show Quotation</h1>
 
 <p><strong>Quote: </strong><%= @quotation.quote %></p>
-<p><strong>Author: </strong><%= @quotation.creator.author if @quotation.creator %></br></br>
+<p><strong>Author: </strong><%= @quotation.creator.author if @quotation.creator %>
+<p><strong>Tags: </strong></br>
+
+<tbody>
+        <% @quotation.tags.each do |tag| %>
+            <tr>
+                <td><%= tag.tag_name if @quotation.tags %></td> 
```

Comment:
remove `if @quotation.tags` inside the `each loop`. You can put it before `each loop`

Reply:
Noted